siphon/simplewebservice/iastate.py

```diff
@@ -173,6 +173,6 @@ def _get_data_raw(self, time, site_id, pressure=None):
                 message += 'for station {stid}'.format(stid=site_id)
             if pressure is not None:
                 message += 'for pressure {pres}'.format(pres=pressure)
-            message = message[:-1] + '.'
+            message = message + '.'
```

Comment:
@zbruick - should this be here, or is this something that accidentally made it in this PR during the merge?

Reply:
This, surprisingly, is a purposeful edit. It's in response to Error #257 in siphon about truncation of the final character in a string. I tested a few different scenarios and weird as it seems, the error seemed to be that the final character was continuously cut off.

Reply:
Awesome, just wanted to be sure. Sometimes little things squeek through a rebase and conflict resolve session.